src/api/app/assets/javascripts/webui/user_profile.js

```diff
@@ -0,0 +1,7 @@
+function placeInvolvement() { // jshint ignore:line
```

Comment:
I know naming things is hard... I understand what this function does, but only after looking at its code.

Do you find something like `moveInvolvementToContainer` clearer?

Reply:
Sure. Renamed to `moveInvolvementToContainer`.